bluebutton-data-pipeline-fhir-load/src/test/resources/logback-test.xml

```diff
@@ -21,7 +21,7 @@
 
 	<!-- Configure the root logger to filter to 'info' and more severe, and 
 		send all events to 'STDOUT'. -->
-	<root level="info">
+	<root level="trace">
```

Comment:
Pretty sure this wasn't necessary. At least... I don't think I've had to add it, myself, anyways.